### ISSUE TITLE
Empty segmentation after anaconda installation

### ISSUE THREAD

**Issue by GiulioBen** [JoHof/lungmask#23]
Dear all, the segmentation was working fine with my images but yesterday I installed a bunch of new packets and anaconda and now the segmentation is no more working.

I guess that it's somehow related with new version of the installed packages but I've not been able to solve the issue.

I tried to remove and reinstall, both in pip3 (/Library/Frameworks/Python.framework/Versions/3.7/bin/pip3) and pip (/Users/Giulio/opt/anaconda3/bin/pip) but the issue is permanent.

```
$ which lungmask
/Users/Giulio/opt/anaconda3/bin/lungmask
```

Using another computer and the same image the segmentation is correct.

Is there a way to check what's going on?

`$ lungmask --modelname LTRCLobes /Users/Giulio/Desktop/test_seg/2_Std_25.nrrd /Users/Giulio/Desktop/test_seg/2_Std_25.seg.nrrd`

Result in my pc (MacOS Mojave) 
![image](https://user-images.githubusercontent.com/38356385/80945212-9aabc200-8deb-11ea-9fe6-0c343b9552c3.png)

Result on my server (Ubuntu 19.04) 
![image](https://user-images.githubusercontent.com/38356385/80945412-0e4dcf00-8dec-11ea-8abd-d0bb62de8d2b.png)

Thank you for the support. 
 If you want I can provide more details on installation/folders etc.

Regards.

**Comment by JoHof** [comment]
Hi Giulio,

given this line:
`lungmask --modelname LTRCLobes /Users/Giulio/Desktop/test_seg/2_Std_25.nrrd /Users/Giulio/Desktop/test_seg/2_Std_25.seg.nrrd`
it looks like you are feeding a segmentation not a proper ct

If you are sure its a correct CT image, does this happen if you install the lungmask package on a completely fresh conda environment as well?

**Comment by JoHof** [comment]
Hi Giulio,

thanks for reportin this. I can confirm this behavious also on linux. Apparantly there went something wrong at the last update. I reverted the master branch back to the previous version 0.2.3. Everything should work as expected. So if you uninstall and reinstall the current master branch it should work. Sorry for that, I will figure out what went wrong!

**Comment by GiulioBen** [comment]
The 2_Std_25.nrrd is a CT, indeed the segmentation works in my server, but not in my laptop.
I tried 

```
conda create newEnv
conda activate newEnv
pip install git+https://github.com/JoHof/lungmask #otherwise it was not working
lungmask ...
```

but the result is the same.
Now i noticed that pip is not linked to the anaconda one but to the default directory (/Library/Frameworks/Python.framework/Versions/3.7/bin/pip).

The i removed the new environment, created a new empty one and reinstalled everything.

```
conda create newEnv
conda activate newEnv
pip install git+https://github.com/JoHof/lungmask #can't used conda install -> gave me "No module named 'numpy' "
conda install numpy
pip install git+https://github.com/JoHof/lungmask
lungmask ...
```

but the result is the same, again an empty segmentation.

It's the first time I use anaconda, but i feel it's not such a good idea to use custom pip packets on it.

**Comment by JoHof** [comment]
Hi Giulio,

I don't thinks it's an issue with anaconda. Custom pip installs usually work well for me in conda. I think it's an issue with scikit learn versions, I am on it. Does it work with the the version thats up now? (0.2.3)

**Comment by GiulioBen** [comment]
Oh yes! that was the problem. I downgraded to 0.2.3 and now it's working!! I didn't noticed I was using different versions!

btw you might add a --version in lungmask just to display the version.

Thank you for the help!! :)

Ciao

**Comment by JoHof** [comment]
You did everything correct! I messed up. I just changed back to 0.2.3. However, I found the bug, it was a datatype issue that happened only when postprocessing volumetric data. I didn't test volumetric data when I changed something. My bad! I will update shortly

> btw you might add a --version in lungmask just to display the version.
Absolutely! Why didn't I do that already?

Thanks for your help!

**Comment by JoHof** [comment]
It's fixed and the --version parameter is also available from 0.2.4 on
